src/api/lib/tasks/dev.rake

```diff
@@ -234,14 +225,42 @@ namespace :dev do
       # Create a request with multiple submit actions and diffs
       Rake::Task['dev:requests:request_with_multiple_submit_actions_builds_and_diffs'].invoke
 
-      # Create a request with a couple of maintenance incident request actions
-      Rake::Task['dev:requests:request_with_maintenance_incident_actions'].invoke
+      # Create a maintenance environment with maintenance requests
+      Rake::Task['dev:maintenance:all'].invoke
 
       # Create a request with a delete request action
       Rake::Task['dev:requests:request_with_delete_action'].invoke
 
       # Create news
       Rake::Task['dev:news:data'].invoke
     end
+
+    desc 'Create Interconnect'
+    task create_interconnect: :development_environment do
+      require 'factory_bot'
+      include FactoryBot::Syntax::Methods
+      create(:remote_project, name: 'openSUSE.org', remoteurl: 'https://api.opensuse.org/public')
+      # The interconnect doesn't work unless we set the distributions
+      FetchRemoteDistributionsJob.perform_now
+    end
+
+    desc 'Create Tumbleweed Repository'
+    task create_tw_repository: :create_interconnect do
+      require 'factory_bot'
+      include FactoryBot::Syntax::Methods
+      interconnect = Project.find_by(name: 'openSUSE.org', remoteurl: 'https://api.opensuse.org/public')
+      create(:repository, name: 'snapshot', project: interconnect, remote_project_name: 'openSUSE:Factory')
```

Comment:
Setting a repository on an interconnect makes little sense. Are we using this somewhere?